dask_expr/reductions.py

```diff
@@ -306,6 +306,10 @@ def reduction_aggregate(cls, df):
 class Min(Max):
     reduction_chunk = M.min
 
+    def _simplify_up(self, parent):
+        if isinstance(parent, Projection):
+            return self.frame[parent.operand("columns")].min(skipna=self.skipna)
```

Comment:
This follows the convention used in other `Reduction` implementations, but as discussed in https://github.com/mrocklin/dask-expr/pull/88#discussion_r1196734567, these can all be written the same way:

```python
    def _simplify_up(self, parent):
        if isinstance(parent, Projection):
            return type(self)(self.frame[parent.operand("columns")], *operands[1:])
```

Reply:
Yeah this needs a rebase when the other PR is merged.

Reply:
Rebased